docs/source/posts/2022/2022-06-20-week1-shivam.rst

```diff
@@ -0,0 +1,47 @@
+First week of coding!
+=====================
+
+.. post:: June 20 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do during the Community Bonding Period?
+----------------------------------------------------
+
+In the community bonding period I met with my mentor Serge Koudoro, along with other mentors in FURY and gsoc contributers.
+We discussed about my project and set up project timeline on github projects section. As my project (glTF Integration)
+and Keyframe animations were related so we discussed on that too.
+
+I read documentation of various glTF loading libraries (``pygltflib``, ``panda3D-gltf`` and ``pyrender``) in python and tried to understand how do they work.
+I started creating a basic glTF loader, it was using dataclasses to store json data. I created polydata for each primitive in a mesh, each polydata will contain the vertices, traingles, normals.
+To apply textures properly, I had to apply texture coordinates to polydata and flip the baseTexture along X axis by 90 degrees.
+
+
+What did you do this week?
+--------------------------
+
+After discussing on pros and cons of various gltf libraries we decided to use ``pygltflib`` to handle json to python dataclass conversion.
+This week I reshaped PR `#600 <https://github.com/fury-gl/fury/pull/600/>`_ to use pygltflib. I also modified the code to handle multiple baseTextures.
```

Comment:
`base textures` maybe

Reply:
this comment has not been addressed

Reply:
I think It should be `baseTextures` as glTF uses the term `baseTexture` in `materials`, So I didn't change that.